tls-client-pelion/mbedtls_mbedos_pelion_config.h

```diff
@@ -0,0 +1,283 @@
+//----------------------------------------------------------------------------
+// The confidential and proprietary information contained in this file may
+// only be used by a person authorised under and to the extent permitted
+// by a subsisting licensing agreement from ARM Limited or its affiliates.
+//
+// (C) COPYRIGHT 2016 ARM Limited or its affiliates.
+// ALL RIGHTS RESERVED
+//
+// This entire notice must be reproduced on all copies of this file
+// and copies of this file may only be made by a person if such person is
+// permitted to do so under the terms of a subsisting license agreement
+// from ARM Limited or its affiliates.
+//----------------------------------------------------------------------------
```

Comment:
Is this the correct copyright? If it is, could you please update the branding and dates?

Reply:
I have copied this file from another ARM repository and I didn't feel authorized to change the copyright notice. If you can confirm, that we are allowed to perform such changes, I will gladly do so.

Reply:
I cannot confirm this, but it would be good to check before merging.